compiler/ballerina-lang/src/main/java/io/ballerina/compiler/api/impl/symbols/BallerinaObjectTypeSymbol.java

```diff
@@ -148,6 +196,33 @@ public BallerinaObjectTypeSymbol(CompilerContext context, ModuleID moduleID, BOb
         return this.typeInclusions;
     }
 
+    public List<TypeSymbol> originalTypeInclusions() {
```

Comment:
Why do we need such a method? There's no distinction to be made in type inclusions.

Reply:
I don't exactly remember what we discussed, but did we not come to a conclusion that the necessary inclusion type info is already there in symbols, and all we have to do is some filtering to capture them from the semantic API side?

Reply:
Made this a private internal method. Used to identify the user-specified type inclusions separately when generating the `.signature()`.